src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -398,7 +398,7 @@ def add_osds(self, drive_group, all_hosts):
 
         if drive_group.hosts(all_hosts)[0] not in [n['name'] for n in current_nodes]:
             pd = { "name": drive_group.hosts(all_hosts)[0],
-                   "storeConfig": { "storeType": drive_group.objectstore }}
+                   "config": { "storeType": drive_group.objectstore }}
```

Comment:
Could you also fix up the comment that is about 20 lines above?

Reply:
fixed